root/etc/apparmor.d/docker_fuse

```diff
@@ -43,9 +43,29 @@ profile docker_fuse flags=(attach_disconnected,mediate_deleted) {
 
   deny /titus/** wl,
 
-/titus/sshd/usr/sbin/run-titus-sshd Cx,
+  # Special sub-profile just for the init script for sshd
+  /titus/sshd/run-titus-sshd Cx,
+  profile run-sshd /titus/sshd/run-titus-sshd {
+    # We trust everything under /titus
+    # No writes though...
+    /titus/** rmlk,
+    /titus/sshd/usr/sbin/sshd ix,
+
+    # You're allowed to do anything with yourself
+    @{PROC}/@{pid}/** rwmlk,
+    @{PROC}/self/** rwmlk,
+
+    /dev/{null,ptmx,tty,urandom,log} rwmlk,
+    /dev/pts/* rwmlk,
+
+    /etc/{gai.conf,group,nsswitch.conf,shadow} rmkl,
+    # The ssh startup script does edit these files if they need modification for sshd to start
+    /etc/{passwd,profile} rwmlk,
+  }
 
-  profile sshd /titus/sshd/usr/sbin/run-titus-sshd {
+  # Once we are running, this is the profile that the sshd binary itself gets
+  /titus/sshd/usr/sbin/sshd Cx,
```

Comment:
Are there any things we can remove from here?

Reply:
Not sure. 
How about I'll deploy this profile on my dev stack, ensure aa is enabled, start with a *blank* profile, try to start a scratch container, ensure it doesnt work, and then slowly add lines until it does work?

In theory the scratch container requires the most out of this `/titus/sshd/usr/sbin/run-titus-sshd` profile, so it should be exercising everything.